app/controllers/products_controller.rb

```diff
@@ -1,30 +1,24 @@
 class ProductsController < ApplicationController
-  before_action :set_product, only: %i[ show edit update destroy ]
+  before_action :set_product, only: %i[ show edit update destroy]
```

Comment:
make it consistent, use both side spacing or remove from both ends (be consistent with your programming style)